src/finn/custom_op/fpgadataflow/hls/thresholding_hls.py

```diff
@@ -52,11 +54,12 @@
 # the ... here can be any shape (representing groups of vectors)
 
 
-class Thresholding_Batch(HLSCustomOp):
+class Thresholding_hls(Thresholding,HLSBackend):
     """Class that corresponds to finn-hls Thresholding_Batch function."""
 
     def __init__(self, onnx_node, **kwargs):
         super().__init__(onnx_node, **kwargs)
+        self.variant = "hls"
```

Comment:
@auphelia , Added this so I can query the variant easily from testing. wasn't sure if you had an alternative for this.